lemminx-liberty/src/test/java/io/openliberty/LibertyDiagnosticTest.java

```diff
@@ -80,32 +80,39 @@ public void testDiagnosticsForInclude() throws IOException {
         String serverXML = String.join(newLine, //
                 "<server description=\"default server\">", //
                 "    <include optional=\"true\" location=\"./empty_server.xml\"/>", //
-                "    <include optional=\"false\" location=\"MISSING FILE\"/>", //
+                "    <include optional=\"true\" location=\"/empty_server.xml\"/>", //
+                "    <include optional=\"true\" location=\"MISSING FILE\"/>", //
```

Comment:
Do you have any tests where the file is missing and the optional attribute is not specified?

Reply:
Will add